src/components/utils.tsx

```diff
@@ -28,3 +29,20 @@ export function addLocatedErrorClassname(
 ) {
   return classnames(classname, locatedErrorClassName);
 }
+
+export function getFileNameByHttpHeaders(headers: Headers) {
+  return getFileNameByContentDisposition(headers.get("Content-Disposition"));
```

Comment:
For consistency with https://github.com/gdsace/molb-libui/blob/3062f8477832030c893dc2c241e1bbcf95a316df/src/components/fileUpload/defaultChild.scss#L49
All the `FileName` can be `Filename` as "filename" is a valid word
```suggestion
  return getFilenameByContentDisposition(headers.get("Content-Disposition"));
```

Reply:
got it. Already update the method's name.